parsec/backend/sequester.py

```diff
@@ -51,13 +52,24 @@ class SequesterServiceAlreadyEnabledError(SequesterError):
     pass
 
 
+class SequesterWrongServiceType(SequesterError):
+    pass
+
+
+class SequesterServiceType(Enum):
+    STORAGE = "storage"
+    WEBHOOK = "webhook"
+
+
 @attr.s(slots=True, frozen=True, repr=False, auto_attribs=True)
 class SequesterService:
     service_id: SequesterServiceID
     service_label: str
     service_certificate: bytes
     created_on: DateTime = attr.ib(factory=DateTime.now)
     disabled_on: Optional[DateTime] = None
+    service_type: SequesterServiceType = SequesterServiceType.STORAGE
```

Comment:
Do we need this ?
I'd rather not have a default value on this given when we build a `SequesterService` object we should entirely define it to avoid silly errors

I guess this is to avoid changes in the tests, can you have a look how much change are requested (if it's too much we can leave as this ^^)